README.md

```diff
@@ -199,7 +208,8 @@ To remove a key from a hash, set the value to `---`. For example, to remove the
 
 ```yaml
 spec/default_facts.yml:
-  ipaddress: '---'
+  networking:
```

Comment:
https://github.com/puppetlabs/pdk-templates/blob/056e50de7da69af4ac876bf9c5e34a752d5fda0c/README.md?plain=1#L198 might want to change `ipaddress` on line 207 to `ip`